docs/cloud-tutorials/account-management.md

```diff
@@ -64,3 +64,43 @@ npx dbos-cloud revoke <token>
 Until they expire or are revoked, refresh tokens can be used to log in to your account.
 Treat them as secrets and keep them safe!
 :::
+
+
+### Organization Management
+
+:::info
+This feature is currently only available to [DBOS Pro or Enterprise](https://www.dbos.dev/pricing) subscribers.
+:::
+
+Organizations allow multiple users to collaboratively manage applications.
+When a user creates an account, they are automatically added to an organization containing only them, where the organization name is the same as their username.
+
+To invite a new user to your organization, run:
+
+```
+npx dbos-cloud org invite
+```
+
+This command retrieves a **single-use** organization secret that expires in 24 hours. To invite multiple users, create multiple secrets.
+
+Using this organization secret, another user can register a new account in your organization by running:
+
+```
+npx dbos-cloud register -u <username> -s <organization-secret>
+```
+
+Alternatively, a user can join your organization by running:
+
+```
+npx dbos-cloud org join <organization-name> <organization-secret>
+```
+
+All users in an organization have full access to its resources, including databases and applications.
+
+You can rename your organization by running:
+
+```
+npx dbos-cloud org rename <current-org-name> <new-org-name>
+```
```

Comment:
This can only be run by org admins, i.e., whoever was the first user in the org

Reply:
Not actually true, I was able to run it with a regular user.

Reply:
That's odd. Let me link you the code offline.

Reply:
ok found the bug